utils/printAssets.js

```diff
@@ -1,33 +1,52 @@
 
 // Prints webpack asset stats
+const fs = require('fs');
 const path = require('path');
 const filesize = require('filesize');
+const gzipSize = require('gzip-size');
 const stripAnsi = require('strip-ansi');
 const logger = require('../cli/logger');
 
-module.exports = (stats) => {
+module.exports = (stats, clientConfig) => {
```

Comment:
Hey @dting Make sure to pass clientConfig in here https://github.com/NYTimes/kyt/blob/master/cli/actions/build.js#L51